index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio: setup and mobile first</title>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <div class="logo-placeholder">
+          <h1>Otmane Ech</h1>
+        </div>
+        <div class="icon">
+          <img src="./assets/Enabled.png" alt="Enabled" />
+        </div>
```

Comment:
- [ ] The hamburger is a button and also part of the nav. The button is to open a mobile menu when clicked on upcoming milestones. For now, kindly wrap the hamburger icon in a button element and wrap the button in a nav element. THis is best for HTML5 semantics.